msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md

```diff
@@ -124,8 +124,131 @@ For the complete app manifest, see [App manifest schema](~/resources/schema/mani
 
+## Universal Actions for search based messaging extensions
+
+The Universal Actions now bring the enhanced user experience available for adaptive cards to messaging extensions. To enable Universal Actions for search based messaging extensions, the app must have the [Schema for Universal Actions](/adaptive-cards/authoring-cards/universal-action-model#schema) along with the following requirements:
+
+1. The app must have a conversation bot defined in their app manifest.
+1. If you already have a conversational bot, the bot must be the same that is used in your messaging extension.
+1. If the card is sent in a group, the users must have group specific permissions to read the card.  Specify `team` and `groupchat` depending on scenarios you plan to support.
+
+**Example**
+
+The following is an example of a json schema with `team` and `groupchat`values:
+
+```json
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.11",
+    "version": "1.0.0",
+    "id": "%MICROSOFT-APP-ID%",
+    "packageName": "com.example.myapp",
+    "bots": [
+        {
+            "botId": "%MICROSOFT-APP-ID-REGISTERED-WITH-BOT-FRAMEWORK%",
+            "scopes": [
+                    "team",
+                    "personal",
+                    "groupchat"
+                ]
+        }
+    ],
+    "composeExtensions": [
+        {
+            "canUpdateConfiguration": true,
+            "botId": "%MICROSOFT-APP-ID-REGISTERED-WITH-BOT-FRAMEWORK%", // Use the same bot as what is specified in the bots section above
+        }
+    ]
+}
+```
+### Automatic refresh for Adaptive Cards in search based MEs
+
+Enable automatic refresh in Adaptive Cards in messaging extensions to ensure users always see up to date data. To enable, define `userIds` array in either `29:<ID>` or `8:orgid:<AAD ID>` format in the `refresh` property.
+
+**Example**
+
+The following is an example of `userIds` array in the `refresh` property:
+
+```json
+    {
+        "type": "AdaptiveCard",
+        "refresh": {
+            "userIds": [
+                "<8:orgId:<AADID>>",
+                "29:<id>"
+            ],
+            "action": {
+                "type": "Action.Execute",
+                "data": {}
+            }
+        },
+        "body": [
+            {
+                "type": "TextBlock",
+                "text": "Hello World!",
+                "wrap": true
+            }
+        ],
+        "actions": [
+            {
+                "type": "Action.Execute",
+                "data": {},
+                "title": "Hello"
+            }
+        ]
+    }
+```
+> [!NOTE]
+> Automatic refresh is enabled for all users in the group chat or channel with more than or equal to 60 users. For conversations (group chat or channel) with more than 60 users, you can provide a manual refresh button or use the refresh button in the message options menu to get the latest result.
```

Comment:
" with **LESS** than or equal to 60 users."

it should not say 'more'

Reply:
Also are we suggesting that the developer can put an Action.Execute button which acts as a way to manual refresh the card? if yes, the language needs to be clearer.